docs/docs/migrate/keptnapp/index.md

```diff
@@ -1,40 +1,49 @@
 # Migrating KeptnApp to KeptnAppContext
 
-The introduction of the `KeptnAppContext` resource
-in Keptn v0.11.0
-requires modifications to existing
+The
+[KeptnAppContext](../../reference/crd-reference/appcontext.md)
+resource is new in the `v1beta1` API included in Keptn v0.11.0.
```

Comment:
I think the last KLT is [klt: v0.9.0](https://github.com/keptn/lifecycle-toolkit/releases/tag/klt-v0.9.0) so the next would be 0.10.0 ... should we try to keep this easy to mantain? So instead of mentioning v1beta1, say something as is introduced from v0.10.0?

Reply:
Oh my!  I was sure we did a 0.10.0 release.  I just spent a significant amount of time sniffing around the web site trying to figure out what the latest release is...  Finally found something that shows metrics-operator 0.9.0 but the most recent lifecycle-operator is 0.8.3!  Maybe the group needs a discussion about how we reference releases and how a user knows which friggin' release they are running?  For now, maybe the docs should just reference the API version?

Reply:
I removed the release number for now.